src/content/en/tools/workbox/v3/modules/workbox-google-analytics.md

```diff
@@ -0,0 +1,164 @@
+which means a Service Worker script can add a fetch handler to detect
+failed requests sent to the Measurement Protocol. It can store these
+requests in IndexedDB and then retry them later once connectivity is
+restored.
+
+Workbox Google Analytics does exactly this. It also also adds fetch
+handlers to cache the
+[analytics.js](/analytics/devguides/collection/analyticsjs/) and
+[gtag.js](/analytics/devguides/collection/gtagjs/)
+scripts, so they can also be run offline. Lastly, when failed requests are
+retried Workbox Google Analytics also automatically sets (or updates) the
+[`qt`](/analytics/devguides/collection/protocol/v1/parameters#qt)
+in the request payload to ensure timestamps in Google Analytics reflect the
+time of the original user interaction.
+
+## Enabling Workbox Google Analytics
+
+To enable Workbox Google Analytics, call the `initialize()` method:
+
+```js
+workbox.googleAnalytics.initialize();
+```
+
+This is the only code that's required to queue and retry failed requests to
+Google Analytics, and it's the simplest way to get Google Analytics working
+offline.
+
+However, if using only the code above, the retried requests are
+indistinguishable from request that succeed on the first try. This means
+you'll receive all the interaction data from offline users, but you won't
+be able to which interactions occurred while the user was offline.
+
+To address this concern you can use one of the configuration options
+described below to modify or annotate the data that gets sent in the
+retried request.
+
+### Modifying what data gets sent
+
+If you want to be able to differentiate retried requests from non-retried
+requests, you can specify either the `parameterOverrides` or `hitFilters`
+[configuration options](../reference-docs/latest/workbox.googleAnalytics#.initialize).
+
+These options let you modify the
+[Measurement Protocol parameters](/analytics/devguides/collection/protocol/v1/parameters)
+that get sent in the retried request. The `parameterOverrides` option
+should be used when you want to set the same value for a particular
+parameter for every retried request. The `hitFilter` option should be used
+in cases where the value of a particular parameter needs to be computed at
+runtime or derived from the value of another parameter.
+
+The examples below show how you'd use both options.
+
+## Examples
+
+### Using a custom dimensions to track online vs. offline interactions
+
+Google Analytics does not have a built-in dimension for online vs. offline
+interactions. However, you can create your own dimension for exactly this
+purpose using a feature called
+[custom dimensions](https://support.google.com/analytics/answer/2709828).
+
+To track requests that were replayed by the service worker using a custom
+dimension with Workbox Google Analytics, follow these steps:
+
+**1)** [Create a new custom dimension](https://support.google.com/analytics/answer/2709829)
```

Comment:
Should these be true ordered lists?

Reply:
The formatting with the indented code block breaks.

Reply:
There is a way to do it, I think you have to indent 4 or 8 spaces?

Reply:
i think it's https://stackoverflow.com/questions/18088955/markdown-continue-numbered-list

Reply:
It doesn't work but I've done it and lets see what happens on prod.